### PR TITLE
updated domain modifier info

### PR DESCRIPTION
**Pull request by anastasiia-fedotova**

### CODE REVIEW

**Review by Alex-302 — APPROVED** [review]

File: docs/general/ad-filtering/create-own-filters.md [hunk 1]
```diff
@@ -331,7 +331,23 @@ Rules with `$denyallow` modifier are not supported by AdGuard for iOS, Safari, a
...
+
+The modifier is a list of one or more expressions separated by `|` symbol, each of which is matched against a domain in a certain way depending on its type (see below).
+
+```
+ domains = ["~"] entry_0 ["|" ["~"] entry_1 ["|" ["~"]entry_2 ["|" ... ["|" ["~"]entry_N]]]]
+ entry_i = ( regular_domain / any_tld_domain / regexp )
+```
+
+* **`regular_domain`** — a regular domain name (`domain.com`). Corresponds the specified domain and its subdomains. It is matched lexicographically.
+* **`any_tld_domain`** — a domain name ending with wildcard character as TLD (`domain.*`).  Corresponds to the specified domain and its subdomains with any TLD. It is matched lexicographically.
```
Comment (sxgunchenko) [inline docs/general/ad-filtering/create-own-filters.md:346]: Actually, the wildcard matches eTLD (effective TLD), not just TLD, at least in CoreLibs. For instance, in `example.co.uk` `co.uk` is eTLD and `uk` is TLD. Not sure if it should be described explicitly but I would propose to fix the term.

@slavaleleka is it true for tsurlfilter?

File: docs/general/ad-filtering/create-own-filters.md [hunk 2]
```diff
@@ -331,7 +331,23 @@ Rules with `$denyallow` modifier are not supported by AdGuard for iOS, Safari, a
 
 #### **`$domain`** {#domain-modifier}
 
-`$domain` limits the rule scope to requests made **from** the specified domains and their subdomains (as indicated by the [Referer](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referer) HTTP header). To add multiple domains to one rule, use the `|` character as a separator.
+`$domain` limits the rule scope to requests made **from** the specified domains and their subdomains (as indicated by the [Referer](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referer) HTTP header). 
+
+###### Syntax
```
Comment (ameshkov) [inline docs/general/ad-filtering/create-own-filters.md:336]:
```suggestion
##### Syntax
```

**Review by ameshkov — CHANGES_REQUESTED** [review]

File: docs/general/ad-filtering/create-own-filters.md (hunk 1 above)
Comment (slavaleleka) [reply docs/general/ad-filtering/create-own-filters.md:346]: for `example.*` in tsurlfilter we check the [public suffix](https://publicsuffix.org/learn/) which is `co.uk` for `example.co.uk`

Comment (sxgunchenko) [reply docs/general/ad-filtering/create-own-filters.md:346]: It is the same. So, any of them can be used here.

File: docs/general/ad-filtering/create-own-filters.md (hunk 2 above)
Comment (slavaleleka) [inline docs/general/ad-filtering/create-own-filters.md:336]: it should be like this
```suggestion
**Syntax**
```

File: docs/general/ad-filtering/create-own-filters.md [hunk 3]
```diff
@@ -331,7 +331,26 @@ Rules with `$denyallow` modifier are not supported by AdGuard for iOS, Safari, a
...
-`$domain` limits the rule scope to requests made **from** the specified domains and their subdomains (as indicated by the [Referer](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referer) HTTP header). To add multiple domains to one rule, use the `|` character as a separator.
+`$domain` limits the rule scope to requests made **from** the specified domains and their subdomains (as indicated by the [Referer](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referer) HTTP header). 
+
+**Syntax**
+
+The modifier is a list of one or more expressions separated by `|` symbol, each of which is matched against a domain in a certain way depending on its type (see below).
+
+```
+ domains = ["~"] entry_0 ["|" ["~"] entry_1 ["|" ["~"]entry_2 ["|" ... ["|" ["~"]entry_N]]]]
+ entry_i = ( regular_domain / any_tld_domain / regexp )
```
Comment (slavaleleka) [inline docs/general/ad-filtering/create-own-filters.md:342]:
```suggestion
domains = ["~"] entry_0 ["|" ["~"] entry_1 ["|" ["~"]entry_2 ["|" ... ["|" ["~"]entry_N]]]]
entry_i = ( regular_domain / any_tld_domain / regexp )
```

File: docs/general/ad-filtering/create-own-filters.md [hunk 4]
```diff
@@ -331,7 +331,26 @@ Rules with `$denyallow` modifier are not supported by AdGuard for iOS, Safari, a
 
 #### **`$domain`** {#domain-modifier}
 
-`$domain` limits the rule scope to requests made **from** the specified domains and their subdomains (as indicated by the [Referer](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referer) HTTP header). To add multiple domains to one rule, use the `|` character as a separator.
+`$domain` limits the rule scope to requests made **from** the specified domains and their subdomains (as indicated by the [Referer](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referer) HTTP header).
```
Comment (slavaleleka) [inline docs/general/ad-filtering/create-own-filters.md:334]:
```suggestion
`$domain` limits the rule scope to requests made **from** the specified domains and their subdomains (as indicated by the [Referer](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referer) HTTP header).
```

File: docs/general/ad-filtering/create-own-filters.md [hunk 5]
```diff
@@ -331,7 +331,26 @@ Rules with `$denyallow` modifier are not supported by AdGuard for iOS, Safari, a
...
+
+The modifier is a list of one or more expressions separated by `|` symbol, each of which is matched against a domain in a certain way depending on its type (see below).
+
+```
+ domains = ["~"] entry_0 ["|" ["~"] entry_1 ["|" ["~"]entry_2 ["|" ... ["|" ["~"]entry_N]]]]
+ entry_i = ( regular_domain / any_tld_domain / regexp )
+```
+
+* **`regular_domain`** — a regular domain name (`domain.com`). Corresponds the specified domain and its subdomains. It is matched lexicographically.
+* **`any_tld_domain`** — a domain name ending with wildcard character as TLD (`domain.*`).  Corresponds to the specified domain and its subdomains with any TLD. It is matched lexicographically.
```
Comment (slavaleleka) [inline docs/general/ad-filtering/create-own-filters.md:346]:
```suggestion
* **`any_tld_domain`** — a domain name ending with wildcard character as a [public suffix](https://publicsuffix.org/learn/),
  e.g. for `example.*` it is `co.uk` in `example.co.uk`. Corresponds to the specified domain
  and its subdomains with any public suffix. It is matched lexicographically.
```

**Review by el-termikael — CHANGES_REQUESTED** [review]

**Review by el-termikael — APPROVED** [review]

**Review by sxgunchenko — APPROVED** [review]
There are plenty of unrelated changes, is it OK?

**Review by ameshkov — APPROVED** [review]

**Pull request merged**